.circleci/config.yml

```
@@ -17,6 +17,7 @@ jobs:
   build_page_files:
     docker:
       - image: mtmiller/octave
+      - image: cimg/python:3.11.0
```

I don't know what happens if you use two images here. Also, I am wondering why we need octave to build the site again. It's been a while since I first set this up, but to my understanding the static site does not need to be aware of the fact, that you can trigger interactive code. So running the code on the site interactively needs octave, but building the static site shouldn't. Or am I missing something? 🤔 

```suggestion
      - image: cimg/python:3.11.0
```

---

Apparently, we do need it, otherwise `List directory /var/lib/apt/lists/partial is missing. - Acquire (13: Permission denied)`

---

Then let's rather fix the permission issue in `cimg/python:3.11.0` than using the container `mtmiller/octave`. Let me try locally how we can install ssh in `cimg/python:3.11.0`.

---

ssh is already installed. We can remove the apt install command entirely.